hnn_core/network.py

```diff
@@ -190,8 +195,17 @@ def __init__(self, params):
             'L5_basket',
             'L5_pyramidal',
         ]
-        self.feedname_list = []  # no feeds defined yet
-        self.feed_times = dict()
+
+        # external drives and biases
+        self.external_drives = dict()
+        self.external_biases = dict()
+        self.drive_event_times = dict()
+
```

Comment:
I suggest a single object. The more attributes you have, the more inconsistencies will arise in the future. I think we can do it with a single object. I still have a hard time figuring out why we need all of ``pos_dict`, `feedname_list`, `_n_common_feeds` etc.